boltdesignsystem/bolt
# Update 5 of Bolt's SVG icon assets to have a more consistent stroke weights

## Jira
http://vjira2:8080/browse/BDS-2051

![image](https://user-images.githubusercontent.com/1617209/75913355-c547c180-5e20-11ea-91ed-6f66ac3a86d8.png)

## Summary
Updates 5 out of the 7 new SVG icons used by Academy to have more consistent stroke weights compared to other existing SVG icons. These include:
- `brand-meteor`
- `brand-molecule`
- `brand-orbit`
- `brand-rocket`
- `brand-triangle`

## How to test
- Confirm updated SVG icons rendering as expected on the `/pattern-lab/?p=components-icon-name-variations` demo page.

I do have a couple of related questions I wanted to raise here:
1. Should we wait for the `brand-atom` and `brand-satelite` icons to also be updated before releasing these? I know they probably aren't getting used however they were part of the original set added...
2. If those two icons aren't getting used, should we (or even, _can we_ / how do we) remove them?
3. The `brand-satelite` icon's name is a typo... 🤦‍♂ what do we do about that?
4. How do we reconcile the `launchpad` and `brand-rocket` icon duplicates?
